<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameSetting</class>
 <widget class="QDialog" name="GameSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游戏设置</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6" stretch="10,3,0,10,1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="4,1">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>15</pointsize>
          </font>
         </property>
         <property name="text">
          <string>单词列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="zongshu_l">
         <property name="text">
          <string>单词总数：NAN</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="wordList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,3,0,5">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>矩形组合</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="zhuhe_cbb"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="reverse_cb">
       <property name="text">
        <string>颠倒布局</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>概率设置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4,2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>雷数</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lei_lin"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>分数设置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>分数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fen_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fen_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fen_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fen_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="fen_5"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>个数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="shu_1"/>
           </item>
           <item>
            <widget class="QLineEdit" name="shu_2"/>
           </item>
           <item>
            <widget class="QLineEdit" name="shu_3"/>
           </item>
           <item>
            <widget class="QLineEdit" name="shu_4"/>
           </item>
           <item>
            <widget class="QLineEdit" name="shu_5"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="cancel_btn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_btn">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
